src/scss/lr7-classic-theme/variables/_list-group.scss

```diff
@@ -0,0 +1,4 @@
+$list-group-active-bg: lighten($brand-primary, 32.5);
```

Comment:
$list-group-active-bg should be the background-color of an active list-group-item, it’s being used to color the bottom border in regular state.